src/layout/layout.js

```diff
@@ -170,23 +175,98 @@
     if (evt.keyCode === this.Keycodes_.ESCAPE &&
         this.drawer_.classList.contains(this.CssClasses_.IS_DRAWER_OPEN)) {
       this.toggleDrawer();
+    } else if (evt.keyCode === this.Keycodes_.SPACE && this.drawer_.classList.contains(this.CssClasses_.IS_DRAWER_OPEN)) {
+      evt.preventDefault();
+      // Send mousedown and mouseup to trigger ripple.
+      var e = new MouseEvent('mousedown');
+      evt.target.dispatchEvent(e);
+      e = new MouseEvent('mouseup');
+      evt.target.dispatchEvent(e);
+      // Send click.
+      evt.target.click();
+    } else if (evt.keyCode === this.Keycodes_.HOME && this.drawer_.classList.contains(this.CssClasses_.IS_DRAWER_OPEN)) {
+      this.drawerMenuItems_[0].focus();
```

Comment:
Why focus on the first item when pressing the `home` key? Is there any particular a11y references to support this behavior?